com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/message/Messages.java

```diff
@@ -1,489 +1,490 @@
+    public static final String ERROR_CREATING_OR_UPDATING_APP = "Error creating or updating application \"{0}\"";
+    public static final String ERROR_UPLOADING_APP = "Error uploading application \"{0}\"";
+    public static final String ERROR_UPLOADING_APP_WITH_DETAILS = "Error uploading application \"{0}\". [{1}] \"{2}\"";
+    public static final String ERROR_UPLOADING_APP_BECAUSE_OF = "Error uploading application \"{0}\": \"{1}\"";
+    public static final String ERROR_RESERVING_ROUTES = "Error reserving routes for application \"{0}\"";
+    public static final String ERROR_CHECKING_UPLOAD_APP_STATUS = "Error checking upload status of application \"{0}\"";
+    public static final String ERROR_SCALING_APP = "Error scaling application \"{0}\"";
+    public static final String ERROR_DETERMINING_ACTIONS_TO_EXECUTE_ON_APP = "Error determining actions to execute on application \"{0}\"";
+    public static final String ERROR_STOPPING_APP = "Error stopping application \"{0}\"";
+    public static final String ERROR_STAGING_APP_1 = "Error staging application \"{0}\"";
+    public static final String ERROR_STAGING_APP_2 = "Staging of application \"{0}\" failed. Download the application logs via the dmol command and check them for more information.";
+    public static final String ERROR_PUBLISHING_PUBLIC_PROVIDED_DEPENDENCIES = "Error publishing public provided dependecies";
+    public static final String ERROR_STARTING_APP_1 = "Error starting application \"{0}\"";
+    public static final String ERROR_STARTING_APP_2 = "Starting of application \"{0}\" failed. Download the application logs via the dmol command and check them for more information.";
+    public static final String ERROR_EXECUTING_APP_1 = "Error executing application \"{0}\"";
+    public static final String ERROR_EXECUTING_APP_2 = "Error executing application \"{0}\": {1}";
+    public static final String ERROR_PREPARING_TO_EXECUTE_TASKS_ON_APP = "Error preparing to execute tasks on application \"{0}\"";
+    public static final String ERROR_EXECUTING_TASK_ON_APP = "Execution of task \"{0}\" failed. Download the application logs \"{1}\", via the dmol command, and check them for more information.";
+    public static final String ERROR_DETECTING_COMPONENTS_TO_UNDEPLOY = "Error detecting components to undeploy";
+    public static final String ERROR_DELETING_IDLE_ROUTES = "Error deleting idle routes";
+    public static final String ERROR_REGISTERING_SERVICE_URLS = "Error registering service URLs";
+    public static final String ERROR_UNREGISTERING_SERVICE_URLS = "Error unregistering service URLs";
+    public static final String ERROR_CREATING_SERVICE_BROKERS = "Error creating service brokers";
+    public static final String ERROR_DELETING_SERVICE_BROKERS = "Error deleting service brokers";
+    public static final String ERROR_UPDATING_SUBSCRIBERS = "Error updating subscribers";
+    public static final String ERROR_NO_SERVICE_URL_SPECIFIED = "No service URL is spe
```

Comment:
Both messages are inconsistent.
Change the second to be : "Cannot retrieve service instance of service with name(optional - decide to keep it or not) \"{0}\""